filament/docs/doxygen/filament.doxygen

```diff
@@ -112,7 +112,15 @@ WARN_LOGFILE           =
 #---------------------------------------------------------------------------
 # Configuration options related to the input files
 #---------------------------------------------------------------------------
-INPUT                  = include backend/include ../libs/filabridge/include/filament
+INPUT                  = ../libs/filabridge/include \
+                         ../libs/geometry/include \
+                         ../libs/gltfio/include \
+                         ../libs/ibl/include \
```

Comment:
are `ibl/include`  and `image/include` supposed to be public headers? I thought they were more for us?
Did they "leak" unto gltfio's public headers?

Reply:
Ah ok, I'll remove ibl.  The image library is fairly well documented so I thought I'd add it but you're probably right, nobody uses it.  I'll remove these two lines.